WordPress/Classes/Networking/ServiceRemoteREST.h

```diff
@@ -1,5 +1,12 @@
 #import <Foundation/Foundation.h>
 
+typedef enum
+{
+    ServiceRemoteRESTApiVersion_1_1 = 1001,
+    ServiceRemoteRESTApiVersion_1_2 = 1002,
+    ServiceRemoteRESTApiVersionDefault = ServiceRemoteRESTApiVersion_1_1,
+} ServiceRemoteRESTApiVersion;
```

Comment:
I'd have no problem with that. Not that it would preven using an arbitrary string, but a typedef could at least hint the developer what's expected.

https://gist.github.com/koke/8f3c1fd3ab9faa53b40d

And you can always add a `NSParameterAssert` if an unexpected version is passed.